### DISCUSSION TITLE
signup error

### DISCUSSION THREAD

**Discussion by yaseressa** [vercel/community#3822] [Account Issue] [answered]
### Question

its my first time signing up and i am already getting We are unable to complete your signup. wth?

**Comment by amyegan** [comment, marked as answer]
Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. Please check for an email with more details about the error and instructions on next steps. For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
